go.mod

```diff
@@ -101,3 +101,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => ./types
```

Comment:
need to remove this replace

Reply:
Just wanted to show that it passes the tests and work fine.
I will open now PR for the types and update to it when merged.